openc3-cosmos-init/plugins/packages/openc3-cosmos-tool-packetviewer/src/router.js

```diff
@@ -26,7 +26,7 @@ import { NotFound } from '@openc3/vue-common/components'
 
 const routes = [
   {
-    path: '/:target?/:packet?',
+    path: '/:target?/:packet?/',
```

Comment:
I'm not seeing changes to the code that makes these URLs, like goto PacketViewer in the Admin Tool.